content/integrations/chef.md

```diff
@@ -5,6 +5,7 @@ kind: integration
 git_integration_title: chef
 newhlevel: true
 updated_for_agent: 5.8.5
+description: "Track Chef client runs: know when they fail, succeed, or make big changes."
```

Comment:
Track Chef client runs including metrics on completion times, analytics on resource changes, and success rates.

Reply:
👍